### DISCUSSION TITLE
Redesigned website homepage feedback

### DISCUSSION THREAD

**Discussion by kvedurmu** [paketo-buildpacks/feedback#13] [General]
Hi! We're looking for feedback on the latest iteration of the Paketo website redesign. @fg-j, @andymoe, and @simonjjones are currently working on just the homepage redesign and any feedback would be appreciated! 

The website changes can be viewed [here](https://paketo.io) and figma mocks (for what this will eventually look like) are available [here](https://www.figma.com/file/vc91zvGMqGxY022FUoOm4S/Paketo_Web_0.01?node-id=57%3A34).

Update - just to clarify, we intend to make these homepage changes live towards the end of the week and intend to use this feedback to help us in future iterations on the homepage design.

**Comment by jromero** [comment]
The empty background and empty space in the tagline area gives me the sense that something is missing.

**Expectation:**

- A pattern or design that is more visually appealing
- Better use of the "above the fold" real estate

**Screenshot:**
![image](https://user-images.githubusercontent.com/475559/122941420-fe1fe880-d33a-11eb-9868-e2dc83e411fb.png)

**Reply by dmikusa** [reply]
All the gray is pretty drab. I'd like to see more color or an interesting pattern too.

**Reply by sophiewigmore** [reply]
+1 @jromero

**Reply by fg-j** [reply]
Fortunately, the colors of the theme are parameterized, so it'll be easy to iterate on switching the colors around 🙂  Once we get the initial redesign merged into main, folks can switch up the colors and put screenshots here if you come up with something you like.

For instance, this update, which I think is festive and great 💅 
![image](https://user-images.githubusercontent.com/21328286/123147006-88944500-d42c-11eb-88cf-cbec046424ad.png)

**Reply by sophiewigmore** [reply]
OMG. Keep it like that!

**Reply by sophiewigmore** [reply]
Anyways, that's good to know. I think I mostly agreed with the empty space part of what Javier said.

**Comment by jromero** [comment]
### Feature Icons are off putting

The entire design of the website has a clean, professional look and feel. The icons appear to be a little different in taste.

1. The lightning bolt looks a little squished
2. The patch looks a little cartoony yet at first it looked like a [CPU chip](https://www.flaticon.com/free-icons/computer-chip).
3. The vine/beanstalk makes little sense and further strays from the clean, professional look.

#### Screenshots

![image](https://user-images.githubusercontent.com/475559/122942429-d41af600-d33b-11eb-8dd8-cc3bb98ca510.png)
![image](https://user-images.githubusercontent.com/475559/122942516-e1d07b80-d33b-11eb-804f-4a7485ae8844.png)

**Comment by jromero** [comment]
## Building animation

I love the building animation. I do have two criticisms though:

1. The `< your app >` looks so much like a placeholder that I literally thought someone forgot to replace a variable. The fact that it's more vibrant should be enough in my opinion to signify its importance. I would suggest that it simply reads as `your app` or something similar (without placeholder notations).
2. Speed: The overall speed is good up until each buildpack is contributing a layer. That part is, for me, painfully slow given that it's repeated 4 times.

**Screenshots:**
![image](https://user-images.githubusercontent.com/475559/122943366-94084300-d33c-11eb-8298-0208216681c2.png)

![image](https://user-images.githubusercontent.com/475559/122943852-fcefbb00-d33c-11eb-89ec-2ba64f1da869.png)

**Comment by jromero** [comment]
### Content 

The content is great!

* 👌 Tagline is spot on!
* ✨ "What are Paketo Buildpacks?" section is clean.
* 🥇 Feature highlights are gold.
* 💖 Love the call out to contribute.

**Comment by jromero** [comment]
### Language icon spacing

The language icon spacing appears to be off. For example, the apache feather is nearly touching the ruby whereas go is keeping some distance from the others.

**Current**

![image](https://user-images.githubusercontent.com/475559/122945555-45f43f00-d33e-11eb-975a-a0430528fc12.png)


**Expected (figma)**

![image](https://user-images.githubusercontent.com/475559/122945639-53112e00-d33e-11eb-9b7f-d8672b719217.png)

**Comment by jromero** [comment]
### Social icons

The social icons are HUGE. I felt like I was about to be attacked. Seriously though, they appear to be bigger than any other icons on the page. If that isn't true, it might be a visual effect by the fact that they have text to the side. Suggestion: make them smaller or remove text.


**Screenshot**

![image](https://user-images.githubusercontent.com/475559/122946079-ac795d00-d33e-11eb-965c-a3f3fa9d25c0.png)

**Comment by dmikusa** [comment]
On my laptop there's is a lot of unused space. Everything seems smushed into the middle 1/3rd of the page. Could things be spaced out a little better on large screens?

<img width="1792" alt="Screen Shot 2021-06-23 at 8 43 03 AM" src="https://user-images.githubusercontent.com/4643262/123098698-6b954d00-d3ff-11eb-9477-e6a79e49d2e9.png">

**Reply by jromero** [reply]
I noticed this too. I then looked at figma and noticed that there's still a missing background image that helps visually but agree that it could be expanded a bit on desktop regardless.

**Comment by dmikusa** [comment]
I found the animation on the menu for the docs to be distracting. Especially if your cursor is at the top of the screen and you're trying to access something at the bottom of the list and you move across the whole list from top to bottom. I think the effect is really cool looking and I sat there playing with it for a bit, but if I'm trying to access the docs that's going to be distracting.


https://user-images.githubusercontent.com/4643262/123099506-3f2e0080-d400-11eb-9cfb-c67fbdad54e8.mov

**Reply by fg-j** [reply]
+10000. we will be rid of this animation when we begin phase 2 and actually redesign the Docs page (the Figma mock has no such animation). We've just re-themed the docs to match the redesigned homepage in phase 1 🙂 

Update: I got rid of the animation because I couldn't un-notice it this morning and it was easy to do.
